### PR TITLE
[W5][W09-1] Lee Shi Mei

### PR DESCRIPTION
**Pull request by shimeilee**

### CODE REVIEW

**Comment by andrewscw** [comment]
Having a HistoryCommand can help users to view clearly what the previous commands they have inputted into the addressbook are, which can prove to help users keep track of what they were doing in case they forgot or make a mistake. Can consider using a stack to store the past commands.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Views all past commands: `history`
+
```
Comment (okkhoy) [inline docs/UserGuide.adoc:160]: :+1: for updating the documentation

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:14]: Don't import everything using the `*`. Import only the classes that are necessary

**Review by okkhoy — COMMENTED** [review]
Useful enhancement here. Good! 
Do remember to update the tests next time. Every new feature should come with (unit) tests else you are at a risk of regression.

**Comment by okkhoy** [comment]
@shimeilee 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**